client/consensus/common/src/import_queue.rs

```diff
@@ -45,6 +45,8 @@ use crate::{
 pub use basic_queue::BasicQueue;
 use sp_consensus::{error::Error as ConsensusError, BlockOrigin, CacheKeyId};
 
+const LOG_TARGET: &str = "import-queue";
```

Comment:
```suggestion
const LOG_TARGET: &str = "sync::import-queue";
```

The good with `sync` is that putting `sync=trace` also gives you these things into the log.